src/UserRepository/PdoDatabaseFactory.php

```diff
@@ -72,14 +72,13 @@ public function __invoke(ContainerInterface $container) : PdoDatabase
             );
         }
 
-        return new PdoDatabase(
-            new PDO(
-                (string) $pdo['dsn'],
-                (string) ($pdo['username'] ?? null),
-                (string) ($pdo['password'] ?? null)
-            ),
-            $pdo,
-            $user
+        $connection = new PDO(
+            (string) $pdo['dsn'],
+            (string) ($pdo['username'] ?? null),
+            (string) ($pdo['password'] ?? null)
         );
+        $connection->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_SILENT);
```

Comment:
We might add `@deprecated` here to enable maintainers to search for deprecations in next versions.
I guess thats what was requested from @froschdesign in https://github.com/mezzio/mezzio-authentication/pull/7#discussion_r504876982

Reply:
Done.